test/integration/tests/button/message.js

```diff
@@ -451,14 +452,18 @@ describe(`paypal button message`, () => {
     });
   });
 
-  describe("modal", () => {
+  describe.only("modal", () => {
```

Comment:
Was this `.only` left in accidentally? 😅

Reply:
😅 indeed, good eye. let me fix that